src/refactor_nrepl/core.clj

```diff
@@ -317,18 +317,27 @@
      (with-meta ns-form (extract-ns-meta (slurp path)))
      (throw (IllegalStateException. (str "No ns form at " path))))))
 
+(def require-lock (Object.))
+
+(defn safe-find-ns [n ignore-error?]
+  (try
+    (locking require-lock
+      (require n))
+    (find-ns n)
+    (catch Throwable e
+      (-> e .printStackTrace)
```

Comment:
Was this left here intentionally?

If so, does it make sense to change the `when-not` to an `if` so we don't print this when we're being asked to ignore errors?